File: examples/preview.go
package main

import "fmt"

type Horizon struct { Name string; Glow int }

func main() {
    // Muted comments, sage strings, and amber functions.
    sky := Horizon{Name: "Afterglow", Glow: 84}
    if sky.Glow > 0 { fmt.Println(sky.Name) }
}

File: examples/Preview.java
record Horizon(String name, int glow) {
    // Types, methods, variables, strings, and control flow.
    public static void main(String[] args) {
        var sky = new Horizon("Afterglow", 84);
        if (sky.glow() > 0) System.out.println(sky.name());
    }
}
